wagtail/models/__init__.py

```diff
@@ -400,6 +400,12 @@ def __init__(self, *args, **kwargs):
     def __str__(self):
         return self.title
 
+    @property
+    def revisions(self):
+        # This acts as a replacement for Django's related manager since we don't
+        # use a GenericRelation/GenericForeignKey.
+        return Revision.page_revisions.filter(object_id=self.id)
```

Comment:
Do we need to use `self.page_ptr_id` instead, if `self` is an instance of the specific page model, or are they the same?